### DISCUSSION TITLE
Message List Component

### DISCUSSION THREAD

**Discussion by petyosi** [petyosi/react-virtuoso#1064] [Show and tell]
The Message List component is the latest addition to the React Virtuoso family. The component comes with:
- imperative data management API, 
- scroll location updates for new messages, 
- optimistic updates, 
- streaming messages, 
- date separators, 
- sticky headers/footers. 

Check the live demo at https://virtuoso.dev/virtuoso-message-list/.

**Comment by leeyuchang** [comment]
Hello.
I have an question about the package update.
Is there any change logs?
I'd like to know what is changed.
![image](https://github.com/user-attachments/assets/9a72319a-0d63-42dd-b7ff-fb44bdb0c9b7)

sincerely

**Comment by petyosi** [comment]
@leeyuchang there's a changelog inside the package: https://www.npmjs.com/package/@virtuoso.dev/message-list?activeTab=code
